brewtils/decorators.py

```diff
@@ -671,21 +671,29 @@ def _initialize_parameter(
         param.default = None
 
     # Now deal with nested parameters
-    if param.parameters:
+    if param.parameters or param.model:
         param.type = "Dictionary"
```

Comment:
If the goal is to find failure criteria first, I would move the `param.type` do to the end of the IF statement next to the `_initialize_parameters` function,